### ISSUE TITLE
Feature request: extended help inside tig

### ISSUE THREAD

**Issue by uffejakobsen** [jonas/tig#534]
I know that tig already got extensive help in form of man-pages and html
But it would be nice if you from within tig could get help on more than the quick reference on keybindings.
For example key 'H' could take you to an searchable info page - similar to the single page html manual

**Comment by jonas** [comment]
Yes, good idea. The simplest might be to use groff to convert the man page to text.

**Comment by IzzySoft** [comment]
I just found tig today (already like it very much ;) Being totally new to it, I really, really miss something like that. Intuitively, I hit the `h` key (as Uffe suggested and as other ncurses tools like `htop` have it), also tried `?` but no dice. Being a good Linux user, `man tig` was the next step, leading me to the homepage and finally to the manual, where I found what I was looking for.

As a reference, especially in the beginning but also for rarely uses commands later, it would be really helpful to have such an "index" bound to an intuitive keystroke. With this issue lying idly for about 18 month, please accept my "bump" :wink:
